src/components/DefaultModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import styled from 'styled-components';
+import MainButton, { Variant } from './MainActionButton';
+
+export const Overlay = styled.div`
+	position: absolute;
+	backdrop-filter: blur(5px);
+	height: 100%;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+`;
+
+export const ButtonField = styled.div`
+	margin-right: 1em;
+	margin-left: 10.5em;
+	width: 100%;
+	display: flex;
+	justify-content: right;
+	grid-gap: 1em;
```

Comment:
Does this actually add a space between the buttons? I didn't think grid-gap worked on flex items

Reply:
That's what I thought, but surprisingly it does 😳